### PR TITLE
john's changes w/ cleaned up bindata

### PR DESCRIPTION
**Pull request by borkmann**
(testing whether Jenkins still has issues with that one)

### CODE REVIEW

**Review by tgraf — APPROVED** [review]

File: tests/14-proxy-ingress.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+sleep 3
+echo " done."
+set -x
+
+cilium endpoint list
+
+cilium policy delete --all
+cat <<EOF | cilium -D policy import -
+[{
+    "endpointSelector": ["id.server"],
```
Comment (jrfastab) [inline tests/14-proxy-ingress.sh]: With the recent changes do we need this here, 

``` "endpointSelector": {"matchLabels":{"id.server":""}}, ```

File: tests/14-proxy-ingress.sh [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    "ingress": [{
+        "fromEndpoints": [
+	    ["reserved:host"], ["id.client"]
```
Comment (jrfastab) [inline tests/14-proxy-ingress.sh]: and same here

``` "fromEndpoints": [ {"matchLabels":{"resreved:host":""}},
                                  {"matchLabels":{"id.client":""}}
```

File: tests/14-proxy-ingress.sh (hunk 1 above)
Comment (borkmann) [reply tests/14-proxy-ingress.sh]: Agree, done.

**Review by jrfastab — APPROVED** [review]
The patches look good to me, thanks for fixing this up.

I'll approve these but is there a way to kick jenkins to run the needed tests? I'm having some issues with my local testing, but it appears unrelated to these patches.

**Comment by tgraf** [comment]
I've force pushed the branch to trigger a new Jenkins run

**Pull request merged**